src/api/base.ts

```diff
@@ -257,6 +257,25 @@ export class BaseAPI {
         }
     }
 
+    async updateCookies(identity: Identity) {
+        const res = await fetch(this.url + 'socket.io/socket.io.js', {
+            method: 'GET',
+            redirect: 'manual',
+            agent: this.agent,
+            headers: {
+                'Connection': 'keep-alive',
+                'Cookie': identity.cookies,
+            }
+        });
+        const header = res.headers.raw()['set-cookie'];
+        if (header !== undefined) {
+            const cookies = header[0].split(';')[0];
+            if (!cookies) { return identity; }
```

Comment:
I mean, why not simply use `if (cookies)`?

Reply:
Exactly